adapters/dmx/dmx.go

```diff
@@ -41,40 +40,21 @@ func UserSellerOrPubId(str1, str2 string) string {
 	return str2
 }
 
-type dmxUserExt struct {
-	Eids      []dmxEids    `json:"eids,omitempty"`
-	Digitrust dmxDigitrust `json:"digitrust,omitempty"`
-}
-type dmxEids struct {
-	Source string `json:"source,omitempty"`
-	Uids   []struct {
-		ID  string `json:"id,omitempty"`
-		Ext json.RawMessage
-	} `json:"uids,omitempty"`
-}
-
-type dmxDigitrust struct {
-	ID   string `json:"id,omitempty"`
-	Keyv int    `json:"keyv,omitempty"`
-}
-
 func (adapter *DmxAdapter) MakeRequests(request *openrtb.BidRequest, req *adapters.ExtraRequestInfo) (reqsBidder []*adapters.RequestData, errs []error) {
 	var imps []openrtb.Imp
 	var rootExtInfo dmxExt
 	var publisherId string
 	var sellerId string
-	var user_ext dmxUserExt
+	var userExt openrtb_ext.ExtUser
 	var anyHasId = false
-	var dmxReq *openrtb.BidRequest = &openrtb.BidRequest{}
+	var dmxReq *openrtb.BidRequest = request
```

Comment:
This will create a new pointer pointing the the same struct that `request` is pointing to. I don't think this is what you want, rather you want something like `var dmxReq openrtb.BidRequest = *request`, or if you need dmxReq to definitely be a pointer:
`var reqCopy openrtb.BidRequest = *request`
`var dmxReq *openrtb.BidRequest = &reqCopy`

Reply:
@hhhjort  thanks for the proposition